src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+/**
+ * Represents a task without any additional info.
+ */
+public class Todo extends Task {
+
+    /**
+     * Initializes a newly created Todo with a description.
+     *
+     * @param desc description of task.
+     */
+    public Todo(String desc) {
+        super(desc);
+    }
+
+    /**
+     * Initializes a newly created Todo with a description and whether it is done.
+     *
+     * @param desc description of task.
+     * @param isDone whether task is done.
+     */
+    public Todo(String desc, boolean isDone) {
+        super(desc, isDone);
+    }
+
+    /**
+     * Sets this task as done.
+     *
+     * @return task set as done.
+     */
+    @Override
+    public Task setDone() {
```

Comment:
Maybe overriding is not necessary as it is same as Task superclass

Reply:
It is actually different as doneTask is assigned a new Todo instead of a new Task.